apps/mull-ui/src/generated/graphql.tsx

```diff
@@ -83,8 +83,8 @@ export type Mutation = {
 
 
 export type MutationAddParticipantToEventArgs = {
-  eventId: Scalars['Float'];
-  userId: Scalars['Float'];
+  eventId: Scalars['Int'];
```

Comment:
You left in your instructions to run `npm run gen-graphql`. You should run it and push the changes it brings